10_integrations/dbt_modal_inference/duck_db_modal_inference.py

```diff
@@ -0,0 +1,276 @@
+# ---
+# cmd: ["modal", "run", "10_integrations/dbt_modal_inference/duck_db_modal_inference.py"]
+# ---
+# # dbt python model with llm inference
+#
+# In this example we demonstrate how you could combine dbt python models with llm
```

Comment:
I think it's a bit confusing to use "models" multiple ways here. Are there alternative ways to describe what DBT is doing? Is it really just providing data models?

Reply:
I agree it's a bit confusing, but calling them sql / python / data models really is the dbt way. I can see if I can make it a bit clearer though